master/buildbot/manhole.py

```diff
@@ -13,6 +13,7 @@
 #
 # Copyright Buildbot Team Members
 from __future__ import print_function
+from builtins import str
```

Comment:
In this instance it is because types.StringType no longer exists in Python3 (the types module is very different on Py3)